javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+var $openModalButton = document.querySelector('button.modal-button');
+var $noButton = document.querySelector('button.no-button');
+var $popUp = document.querySelector('div.position-fixed');
+
+$openModalButton.addEventListener('click', function (event) {
+  $popUp.className = 'row center position-fixed tint';
+});
+
+$noButton.addEventListener('click', function (event) {
+  $popUp.className += 'display-none';
```

Comment:
Does this actually work? I would think the last class would end up being `tintdisplay-none` which would break everything 🤔